vendor/wax.cartodb.js

```diff
@@ -3359,7 +3359,7 @@ wax.g.connector.prototype.getTileUrl = function(coord, z) {
 
     x = (x < 0) ? (coord.x % mod) + mod : x;
 
-    if (y < 0) return this.options.blankImage;
+    if (y < 0 || y > (mod - 1)) return this.options.blankImage;
```

Comment:
I guess we could use `y >= 0` too.
I'm not a big fan of comments... but perhaps it makes sense to add one here (along [these lines](https://github.com/CartoDB/wax/pull/11#issue-205841475)). What do you think?